CHANGES/8299.packaging.rst

```diff
@@ -0,0 +1,2 @@
+Added an 'internal' pytest marker for tests which should be skipped
```

Comment:
It might make sense to either symlink a "contrib" change note or make a separate one, with a usage example and a code snippet.